study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/task/BrainConsentVersion2.java

```diff
@@ -125,8 +125,10 @@ public void run(Handle handle) {
 
         //remove consent icons
         List<Long> iconIds = helper.findSectionIconIdByActivity(activityId);
-        helper.deleteActivityIcons(Set.copyOf(iconIds));
-        LOG.info("deleted form section icons for consent");
+        if (!iconIds.isEmpty()) {
```

Comment:
change to make it backward compatible if we ever re-initialize the study in lower regions.